### PR TITLE
RegEx checkbox and exact match now attached to form fields

### PR DESCRIPTION
**Pull request by ifarzana**

### CODE REVIEW

File: views/campaign-page.twig [hunk 1]
```diff
@@ -38,16 +38,24 @@ This is the template for the campaign page
                 <div class="XiboFilter well">
                     <div class="FilterDiv" id="Filter">
                         <form class="form-inline">
-                            {% set title %}{% trans "Name" %}{% endset %}
-                            {{ inline.input("name", title) }}
+
+                            <div class="form-group ">
+                                <label class="control-label" title="" for="name" accesskey="">Name</label>
```
Comment (dasgarner) [inline views/campaign-page.twig]: this needs to be translated

File: views/dataset-page.twig [hunk 2]
```diff
@@ -24,8 +24,19 @@
                 <div class="XiboFilter well">
                     <div class="FilterDiv" id="Filter">
                         <form class="form-inline" onsubmit="return false">
-                            {% set title %}{% trans "Name" %}{% endset %}
-                            {{ inline.input("dataSet", title) }}
+
+                            <div class="form-group ">
+                                <label class="control-label" title="" for="dataSet" accesskey="">Name</label>
```
Comment (dasgarner) [inline views/dataset-page.twig]: trans

File: views/display-page.twig [hunk 3]
```diff
@@ -33,8 +33,17 @@
                                     {% set title %}{% trans "ID" %}{% endset %}
                                     {{ inline.number("displayId", title) }}
 
-                                    {% set title %}{% trans "Name" %}{% endset %}
-                                    {{ inline.input("display", title) }}
+                                    <div class="form-group ">
```
Comment (dasgarner) [inline views/display-page.twig]: trans

File: views/displaygroup-page.twig [hunk 4]
```diff
@@ -26,8 +26,17 @@
                             {% set title %}{% trans "ID" %}{% endset %}
                             {{ inline.input("displayGroupId", title) }}
 
-                            {% set title %}{% trans "Name" %}{% endset %}
-                            {{ inline.input("displayGroup", title) }}
+                            <div class="form-group ">
+                                <label class="control-label" title="" for="displayGroup" accesskey="">Name</label>
```
Comment (dasgarner) [inline views/displaygroup-page.twig]: trans

File: views/displayprofile-page.twig [hunk 5]
```diff
@@ -23,8 +23,18 @@
                 <div class="XiboFilter well">
                     <div class="FilterDiv" id="Filter">
                         <form class="form-inline">
-                            {% set title %}{% trans "Name" %}{% endset %}
-                            {{ inline.input("displayProfile", title) }}
+
+                            <div class="form-group ">
+                                <label class="control-label" title="" for="displayProfile" accesskey="">Name</label>
```
Comment (dasgarner) [inline views/displayprofile-page.twig]: trans

File: views/layout-page.twig [hunk 6]
```diff
@@ -25,15 +25,22 @@
                                     {% set title %}{% trans "ID" %}{% endset %}
                                     {{ inline.number("layoutId", title) }}
 
-                                    {% set title %}{% trans "Name" %}{% endset %}
-                                    {{ inline.input("layout", title) }}
+                                    <div class="form-group ">
+                                        <label class="control-label" title="" for="layout" accesskey="">Name</label>
```
Comment (dasgarner) [inline views/layout-page.twig]: trans

File: views/library-page.twig [hunk 7]
```diff
@@ -32,15 +32,22 @@
                             {% set title %}{% trans "ID" %}{% endset %}
                             {{ inline.number("mediaId", title) }}
 
-                            {% set title %}{% trans "Name" %}{% endset %}
-                            {{ inline.input("media", title) }}
+                            <div class="form-group ">
+                                <label class="control-label" title="" for="media" accesskey="">Name</label>
```
Comment (dasgarner) [inline views/library-page.twig]: trans

File: views/log-page.twig [hunk 8]
```diff
@@ -107,9 +107,17 @@
                                     ] %}
                                     {{ inline.dropdown("displayId", "single", title, "", null, "displayId", "display", "", "pagedSelect", "", "d", "", attributes) }}
 
-
-                                    {% set title %}{% trans "Display Name" %}{% endset %}
-                                    {{ inline.input("display", title) }}
+                                    <div class="form-group ">
+                                        <label class="control-label" title="" for="display" accesskey="">Display Name</label>
```
Comment (dasgarner) [inline views/log-page.twig]: trans

Comment (ifarzana) [reply views/log-page.twig]: lol

**Pull request merged**